### DISCUSSION TITLE
Favicon indicates status of SQL Lab query

### DISCUSSION THREAD

**Discussion by drummerwolli** [apache/superset#25754] [Ideas]
A SQL Lab query can run long. Usually you are executing a query and have it then running in the background, while you do something else. If you are using the same browser window but a different tab than where Superset is, you need to go to the Superset tab first to see if the query finished or not.

The idea is to show with the favicon what the status of the query is:

no query issued yet:

![image](https://github.com/apache/superset/assets/3867569/2717a443-bb76-4c94-a248-42736001a3d9)

Query still running:

![image](https://github.com/apache/superset/assets/3867569/739a5852-07c8-499b-b29b-de456e0cd817)

Query successful:

![image](https://github.com/apache/superset/assets/3867569/e506d13e-d2c3-4233-a4e3-b3b720bf7669)

Query failed:

![image](https://github.com/apache/superset/assets/3867569/7121a08e-292a-4bc5-be9c-738928b0daba)

all other pages we would still show the current Favicon:

![image](https://github.com/apache/superset/assets/3867569/194747fa-493d-4efa-916f-5190dad3b2e7)

(the examples shown are just for illustration purposes of the feature. I'm not a designer nor a UX expert. I dont mind how this looks as long as the query status is readable from the favicon.

Basically saving the user time and head space to switch between tabs (potentially also loosing context in between, because you're distracted by something else as well)

It's a little thing, but I would highly appreciate it. It's implemented in some other tools with long-running tasks (CI/CD platforms, etc.) and there i found myself relying on it a lot.

I was also wondering the same kind of indicator for dashboards and their status of loading the data, but wanted to start with SQL Lab first.
